src/CTA.Rules.Actions/MethodDeclarationActions.cs

```diff
@@ -23,5 +23,22 @@ public class MethodDeclarationActions
             };
             return AddComment;
         }
+
+        public Func<SyntaxGenerator, MethodDeclarationSyntax, MethodDeclarationSyntax> GetAddExpressionAction(string expression)
+        {
+            // TODO: This will add an expression at the bottom of a method body, in the future we should add granularity for where to add the expression within a method body
+            Func<SyntaxGenerator, MethodDeclarationSyntax, MethodDeclarationSyntax> AddExpression = (SyntaxGenerator syntaxGenerator, MethodDeclarationSyntax node) =>
+            {
+                StatementSyntax statementExpression = SyntaxFactory.ParseStatement(expression);
+                if(!statementExpression.FullSpan.IsEmpty)
+                {
+                    BlockSyntax nodeBody = node.Body;
+                    nodeBody = nodeBody.AddStatements(statementExpression);
+                    node = node.WithBody(nodeBody).NormalizeWhitespace();
+                }
+                return node;
+            };
+            return AddExpression;
+        }
```

Comment:
Since this adds an expression to the end of a function, would it make sense to rename this action to `AppendExpression`? If we have other actions that behave similarly, then I think it's better to stick with the convention. If not, renaming might make sense since implementing the ability to specify where to insert an expression is a non-trivial task.

Reply:
Agreed, I went ahead and renamed it and also renamed the class action AddConstructorExpression to AppendConstructorExpression also to follow this schema.